docs/manifest.yaml

```diff
@@ -24,7 +24,7 @@ spec:
       id: tracing
     - displayName: API Gateway
       id: api-gateway
-    - displayName: Authorization and Authentication
-      id: authorization-and-authentication
+    - displayName: Security
+      id: security
```

Comment:
L.G. suggested to change the order of the components and move the Security topic at the top of the components' list - to important to be at the bottom.